OpenSim/Simulation/Model/ContactMesh.cpp

```diff
@@ -145,7 +148,7 @@ void ContactMesh::generateDecorations(bool fixed, const ModelDisplayHints& hints
         const auto& X_BF = getFrame().findTransformInBaseFrame();
         const auto& X_FP = getTransform();
         const auto X_BP = X_BF * X_FP;
-        geometry.push_back(SimTK::DecorativeMeshFile(get_filename())
+        geometry.push_back(SimTK::DecorativeMesh(*_decorativeGeometry)
```

Comment:
Ah I see...but as is, the model display hints do not affect the `SimTK::DecorativeMesh` that gets sent to the visualizer. However, as this may change if you introduce a display hint for the overall size of the mesh, then it's fine with me to leave in the extra copying.

What happens when `get_show_contact_geometry() == true` but the Force `isDisabled = true`? In this case, it seems like `_decorativeGeometry` will be null, but this code will still try to use it.